bedrock/firefox/templates/firefox/desktop/trust.html

```diff
@@ -48,25 +48,28 @@
       </p>
 
       <ul class="feature-list">
-        <li id="privacy-dnt">
-          <h3>{{ _('Do Not Track') }}</h3>
+        {% if l10n_has_tag('tracking_protection') %}
```

Comment:
Note: we seem to use a convention of using hyphens `-` in waffle switches looking at other pages, as opposed to underscores `_`.